src/computation/logic/district-merging.ts

```diff
@@ -166,10 +44,10 @@ function groupVotes(
             if (!groupedVotes.has(newName)) {
                 groupedVotes.set(newName, new Map<string, Votes[]>());
             }
-            let votes = groupedVotes.get(newName);
-            if (votes !== undefined) {
-                votes = mapAdd(votes, currentVote.party, currentVote);
-                groupedVotes.set(newName, votes!);
+            let voteMap = groupedVotes.get(newName);
+            if (voteMap !== undefined) {
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5976811)